tests/conftest.py

```diff
@@ -137,7 +199,8 @@ def event_queues(app, event_entrypoints):
 def base_app():
     """Flask application fixture without InvenioStats."""
     from invenio_stats.config import STATS_EVENTS
-    app_ = Flask('testapp')
+    instance_path = tempfile.mkdtemp()
```

Comment:
The instance directory is not removed at the end.

Reply:
ah right, fixed.